core/state/journal.go

```diff
@@ -42,6 +50,9 @@ type journalEntry interface {
 type journal struct {
 	entries []journalEntry         // Current changes tracked by the journal
 	dirties map[common.Address]int // Dirty accounts and the number of changes
+
+	validRevisions []revision
+	nextRevisionId int
```

Comment:
These two fields are not deep-copied in `copy` function

Reply:
Good catch, thanks!